### PR TITLE
[DOC] Update installation instructions

### LINKED ISSUE #323: [ENH] Update installation instructions
### Is there an existing issue for this?

- [X] I have searched the existing issues

### New feature

- [ ] Now that we have a PyPI package, we should add `pip install nipoppy` as the main installation method
- [ ] We can also add a PyPI badge to the README

### Unclear documentation

_No response_

### PR DESCRIPTION
**Pull request by mathdugre**
- Closes #323 

Changes proposed in this pull request:

- Update installation instruct since we now release on PyPI.
- Simplify some installation steps to be more welcoming to new users.

## Checklist (for reviewers)
_This section is for the PR reviewer_

- [x] PR has an interpretable title with a prefix (e.g. `[BUG]`, `[DOC]`, `[ENH]`, `[MAINT]`)\
Refer to [NumPy Development Guide](https://numpy.org/doc/stable/dev/development_workflow.html#writing-the-commit-message) for a full list
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [ ] ~~Tests pass~~
- [x] Checks pass

For new features:
- [ ] ~~Tests have been added~~

For bug fixes:
- [ ] ~~There is at least one test that would fail under the original bug conditions~~

### CODE REVIEW

File: nipoppy_cli/docs/source/installation.md [hunk 1]
```diff
@@ -35,79 +35,13 @@ Activate the environment, e.g. by running:
...
----
-class: dropdown
----
-If you are using one of the [Compute Canada/Digital Research Alliance of Canada](https://docs.alliancecan.ca/wiki/Technical_documentation) {term}`HPC` systems, you should instead use `virtualenv`:
-```{code-block} console
-$ virtualenv --no-download nipoppy_env
-```
-
-See the [Compute Canada wiki](https://docs.alliancecan.ca/wiki/Python#Creating_and_using_a_virtual_environment) for more information.
-````
```
[the remark this answers is not in the document]
Comment (michellewang) [reply nipoppy_cli/docs/source/installation.md:61]: Also in general I'm a bit unsure about removing the `venv` instructions completely and pointing users to the Python docs. @nikhil153 do you have thoughts on this?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Nipoppy
 
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8084759.svg)](https://doi.org/10.5281/zenodo.8084759)
+[![PyPI - Version]([https://img.shields.io/pypi/v/nipoppy])](https://pypi.org/project/nipoppy/)
```
Comment (michellewang) [inline README.md:4]: Seems to be causing the docs build to fail with error:
```
/home/docs/checkouts/readthedocs.org/user_builds/nipoppy/checkouts/326/README.md:4: WARNING: image file not readable: ../../../%5Bhttps:/img.shields.io/pypi/v/nipoppy%5D [image.not_readable]
```

Not sure why, and previously it was fine...

Comment (mathdugre) [reply README.md:4]: I think made an error in the link. It has extras square brackets around the url.

File: nipoppy_cli/docs/source/installation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikhil153) [reply nipoppy_cli/docs/source/installation.md:61]: Does this mean we only have `conda` installation information? I am okay removing lines related to Compute Canada, but we should have both `conda` and `venv` create/install environment instructions.

[the remark this answers is not in the document]
Comment (mathdugre) [reply nipoppy_cli/docs/source/installation.md]: I added back the `venv` installation info 👍

**Comment by mathdugre** [comment]
waiting on #339 to be merge as it will affect the installation procedure

**Review by michellewang — APPROVED** [review]
👩‍🍳

**Pull request merged**